core/domain/auth_services.py

```diff
@@ -282,3 +288,50 @@ def revoke_super_admin_privileges(user_id: str) -> None:
         user_id: str. The Oppia user ID to revoke privileges from.
     """
     firebase_auth_services.revoke_super_admin_privileges(user_id)
+
+
+def get_csrf_secret_value() -> str:
+    """Returns the csrf secret value if it exists otherwise initializes it.
+
+    Returns:
+        str. Returns the csrf secret value.
+    """
+    memcached_items = caching_services.get_multi(
```

Comment:
If you do this then it means you'll need to be very sure to flush the memcache immediately after release happens so that a new value definitely gets generated and the memcache doesn't retain the CSRF value from before.

Since this is an instruction for release coordinators, please follow the [wiki page](https://github.com/oppia/oppia/wiki/Testing-jobs-and-other-features-on-production) for "server actions" to make sure it happens.

Reply:
Got it, I have mentioned this in the "steps" section of the testing doc.